packages/foam-vscode/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to the "foam-vscode" extension will be documented in this fi
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## Unreleased
+
+- Unknown Foam template variables are now ignored by Foam.
```

Comment:
I like this, I wonder whether it might become source of conflicts when rebasing/merging feature branches, but I like the approach otherwise